<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Pomodoro</class>
 <widget class="QWidget" name="Pomodoro">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>406</width>
    <height>238</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Pomodoro</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>20</y>
     <width>168</width>
     <height>215</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QLabel" name="curTime">
      <property name="font">
       <font>
        <pointsize>48</pointsize>
       </font>
      </property>
      <property name="text">
       <string>00:00</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QRadioButton" name="opt_pomodoro">
        <property name="text">
         <string>&amp;Pomodoro</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="opt_shortbreak">
        <property name="text">
         <string>Shor&amp;t Break</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="opt_longbreak">
        <property name="text">
         <string>&amp;Long Break</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QPushButton" name="btnControl">
      <property name="text">
       <string>Start</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
